opentelekomcloud/services/iam/data_source_opentelekomcloud_identity_user_v3.go

```diff
@@ -92,13 +89,22 @@ func dataSourceIdentityUserV3Read(_ context.Context, d *schema.ResourceData, met
 
 	d.SetId(user.ID)
 	mErr := multierror.Append(
-		d.Set("default_project_id", user.DefaultProjectID),
 		d.Set("domain_id", user.DomainID),
 		d.Set("enabled", user.Enabled),
 		d.Set("name", user.Name),
 		d.Set("password_expires_at", user.PasswordExpiresAt.Format(time.RFC3339)),
 	)
 
+	mfa, err := users.ShowUserMfaDevice(client, user.ID)
+	switch err.(type) {
+	case golangsdk.ErrDefault403:
+		break
```

Comment:
add printf about permissions, not enough only mention that in doc

Reply:
done